.github/workflows/buildAndPublish.yml

```diff
@@ -0,0 +1,57 @@
+name: Build and Publish @extrahorizon/exh-cli
+
+on:
+  push:
+    branches:
+      - master
+      - dev
+
+jobs:
+  build-and-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Extract version
+        id: extract_version
+        uses: Saionaro/extract-package-version@v1.0.6
+      - name: Print version
+        run: echo ${{ steps.extract_version.outputs.version }}
+      - name: Create .npmrc
+        run: |
+          echo "@extrahorizon:registry=https://npm.pkg.github.com" > .npmrc
+          echo "//npm.pkg.github.com/:_authToken=${{secrets.GITHUB_TOKEN}}" >> .npmrc
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          registry-url: https://npm.pkg.github.com/
+          scope: '@extrahorizon'
+      - run: yarn
+      - run: yarn lint
+      - run: yarn build
+      - run: yarn test
```

Comment:
Everything up until here is exactly the same as `ci-development` or `ci-production`